src/complex-list-field/complex-list-field.component.html

```diff
@@ -64,7 +64,7 @@
         <!-- ADD-FIELD-FROM-SCHEMA, UP/DOWN and DELETE buttons for each row group -->
         <tr *ngIf="values.size > 0">
           <td class="button-holder">
-            <add-field-dropdown [fields]="keys[i]" (onFieldAdd)="onFieldAdd(i, $event)" [schema]="schema.items.properties"></add-field-dropdown>
+            <add-field-dropdown [fields]="keys[i]" (onFieldAdd)="onFieldAdd(i, $event)" [schema]="schema.items.properties">+</add-field-dropdown>
```

Comment:
I am not sure where the `+` is added in the UI. Cannot see the difference from before ;)